src/components/Sum.jsx

```diff
@@ -1 +1,27 @@
-// Don't forget to add default values an check types of props
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a, b }) => (
```
When there are less than 3 arguments you can write them in one line